Userland/Services/AudioServer/ClientConnection.h

```diff
@@ -49,7 +49,7 @@ class ClientConnection final : public IPC::ClientConnection<AudioClientEndpoint,
     virtual Messages::AudioServer::GetPlayingBufferResponse get_playing_buffer() override;
     virtual Messages::AudioServer::GetMutedResponse get_muted() override;
     virtual void set_muted(bool) override;
-    virtual void set_sample_rate(u32 sample_rate) override;
+    virtual void set_sample_rate(u16 sample_rate) override;
```

Comment:
You changed it from `u32` to `u16` here, but...

Reply:
Ohh yeah, I remember this. That was the reason I had a merge conflict 🐞 

I'll fix it (: